portfolio/src/main/webapp/script.js

```diff
@@ -43,11 +43,48 @@ function displayAbout() {
     });
 }
 
+function displayGame() {
+    const title = document.getElementById('content-title');
+    title.innerHTML = "Let's play a fun game (well, at least I find it fun <i class=\"em em-smile\" aria-role=\"funny\" aria-label=\"Smile\"></i>)";
+    $(function(){
+        $("#content-panel").load("main_page_clickables/game.html"); 
+    });   
+}
+
+function displayCountry() {
+    console.log("Getting the country name country");
+
+    const responsePromise = fetch('/data');
+
+    responsePromise.then(handleResponse);
+}
+
+function handleResponse(response) {
```

Comment:
Looks good!

Once you've got the hang of handling promises and connecting them together, it'll be simpler to write this as a single statement to make it less verbose.

Reply:
I look forward to that. As of now, I find it very confusing the compactness of those arrow functions.